libgc/mark.c

```diff
@@ -1191,11 +1191,11 @@ void GC_mark_init()
  * Should only be used if there is no possibility of mark stack
  * overflow.
  */
-void GC_push_all(bottom, top)
-ptr_t bottom;
-ptr_t top;
+void GC_push_all(void* void_bottom, void* void_top)
```

Comment:
I understand you might find it very ugly (we all do), but keep the old C parameters declaration.

Reply:
The types, or syntax, or both?

Reply:
This matches mainline and unity fwiw.

Reply:
```
void GC_push_all(bottom, top)
void* bottom;
void* top;
```

Reply:
The types matter, a little, but I can undo that if you insist. The syntax does not matter at all, looks fine either way.

Reply:
Ok, no problem. It was char* before, is the real point.